BUILD

```diff
@@ -4,6 +4,8 @@ load("//base:distro.bzl", "DISTROS", "LANGUAGE_DISTROS")
 
 package(default_visibility = ["//visibility:public"])
 
+default_distro = "debian12"
```

Comment:
@thesayyn is this bazel normal convention? for variable naming?

Reply:
constants are usually uppercase.

Reply:
hah yeah thanks, I eventually found some docs.